HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewController.swift

```diff
@@ -14,36 +14,47 @@ class UserFavouritesScreenViewController: UIViewController, ViewModelProtocol {
     @IBOutlet weak private var userImage: UIImageView!
     @IBOutlet weak private var favouritesTitleLabel: UILabel!
     @IBOutlet weak private var favouritesCollectionView: UICollectionView!
+    @IBOutlet weak private var noFavouritesLabel: UILabel!
     
     // MARK: Variables
     
     private lazy var userFavouritesScreenViewModel = UserFavouritesScreenViewModel(userFavouritesScreenRepository: UserFavouritesScreenRepository(),
                                                                                    delegate: self)
-    
     // MARK: Functions
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setUpCollectionView()
         setUpScreen()
+        noFavouritesLabel.isHidden = !userFavouritesScreenViewModel.showFavouritesText
     }
     
     func setUpScreen() {
         userFavouritesScreenViewModel.fetchAllMarvelDataFromCoreData()
         favouritesTitleLabel.text = userFavouritesScreenViewModel.favouritesScreenTitle
-        
+        if userFavouritesScreenViewModel.marvelDataListCount > 0 {
+            noFavouritesLabel.isHidden = true
+        }
     }
+    
     func reloadView() {
         favouritesCollectionView.reloadData()
     }
     
+    func startLoading() {
```

Comment:
please put these functions in an extension of UserFavouritesScreenViewController like you did for home screen